robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -72,6 +72,10 @@ public class ShadowAccessibilityNodeInfo {
   private static final int EDITABLE_MASK = 0x00000080;
   
   private static final int TEXT_SELECTION_SETABLE_MASK = 0x00000100;
+  
+  private static final int CLEAR_A11YFOCUSABLE_MASK = 0x00000200;
```

Comment:
Any reason we didn't just use the values in AccessibilityNodeInfo? The action constants seem all to be masks.